Makefile

```diff
@@ -1,26 +1,45 @@
-.PHONY: check check-core dry-core-admin core-admin dry-core core
+.PHONY: check check-core check-services dry-core-admin dry-services-admin core-admin services-admin dry-core core dry-services services all-admin all
 
-check: check-core
+check: check-core check-services
 	@echo "Service config check: PASS"
 
 check-core:
 	core-services/_hack/validate-core-services.sh core-services
 	@echo "Core service config check: PASS"
 
+check-services:
+	core-services/_hack/validate-core-services.sh services
+	@echo "Service config check: PASS"
+
 # applyconfig is https://github.com/openshift/ci-tools/tree/master/cmd/applyconfig
 
 dry-core-admin:
 	applyconfig --config-dir core-services --level=admin
 
+dry-services-admin:
+	applyconfig --config-dir services --level=admin
+
 core-admin:
 	applyconfig --config-dir core-services --level=admin --confirm=true
 
+services-admin:
+	applyconfig --config-dir services --level=admin --confirm=true
+
 dry-core:
 	applyconfig --config-dir core-services
 
+dry-services:
+	applyconfig --config-dir services
+
 core:
 	applyconfig --config-dir core-services --confirm=true
 
+services:
+	applyconfig --config-dir services --confirm=true
+
+all-admin: core-admin services-admin
+all: core-admin core services-admin services
```

Comment:
These can be reversed and moved above line 3 so `make` == `make all`.

Reply:
Done.